akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -859,13 +906,15 @@ object PartitionHub {
 /**
  * INTERNAL API
  */
-private[akka] class PartitionHub[T](partitioner: () ⇒ (Array[Long], T) ⇒ Long, startAfterNbrOfStreams: Int, bufferSize: Int)
+private[akka] class PartitionHub[T](partitioner: () ⇒ (Array[Long], T) ⇒ Long, startAfterNbrOfConsumers: Int, bufferSize: Int)
```

Comment:
Some fail-fast requires here on non negative buffer size and maybe nbr of consumers?